package.json

```diff
@@ -53,7 +53,9 @@
         "*.{js,ts,css,md}": "prettier --write --ignore-unknown"
     },
     "dependencies": {
+        "@graphql-tools/utils": "^8.8.0",
         "graphql": "^16.5.0",
+        "graphql-tools": "^8.3.0",
         "ioredis": "^5.0.5"
```

Comment:
Are the dependencies for `graphql-tools/utils` and `graphql-tools` needed? I didn't see them used anywhere.

Reply:
these were from an initial attempt to parse directives. removed